### PR TITLE
Feature/ees 687 related information

### PR DESCRIPTION
**Pull request by bennettstuart**
Fixes from bugs found in testing...
https://alpha-dfe-data.atlassian.net/browse/EES-687

The `<Link .. >` component was a router link component and so was routing to `/{url} in the app, as opposed to link externally to the url.

The methodology href updated to link to `/methodology/{methodologyId}` in the app, the same as the methodology link at the bottom of the page

### CODE REVIEW

File: src/explore-education-statistics-admin/src/modules/find-statistics/components/EditableLink.tsx [hunk 1]
```diff
@@ -1,23 +1,58 @@
...
+      rel="noopener noreferrer"
+      target="_blank"
+    >
+      {children}
+    </a>
   ) : (
     <div>
       <div>
-        <Link {...props} />
+        <a
```
Comment (duncan-at-hiveit) [inline src/explore-education-statistics-admin/src/modules/find-statistics/components/EditableLink.tsx]: You may wanna reuse the "Link" Component here?

File: src/explore-education-statistics-common/src/services/publicationService.ts [hunk 2]
```diff
@@ -37,6 +37,7 @@ export interface Publication {
   contact: PublicationContact;
   methodology: {
     id: string;
+    title: string;
```
Comment (duncan-at-hiveit) [inline src/explore-education-statistics-common/src/services/publicationService.ts]: We have a structure called IdTitlePair that you could probably reuse here

File: src/explore-education-statistics-admin/src/modules/find-statistics/components/EditableLink.tsx (hunk 1 above)
Comment (bennettstuart) [reply src/explore-education-statistics-admin/src/modules/find-statistics/components/EditableLink.tsx]: The Link component is of `'react-router-dom'`
https://reacttraining.com/react-router/web/api/Link

AFAIK doesn't support linking like an anchor with a href, only within the app routes

I've adapted `@admin`'s Link to act differently in the case where a `href` is supplied and `to == false`

File: src/explore-education-statistics-common/src/services/publicationService.ts (hunk 2 above)
Comment (bennettstuart) [reply src/explore-education-statistics-common/src/services/publicationService.ts]: unfortunately `IdTitlePair` is located in `@admin` and here we're making changes in the `@common` sub-project

**Comment by bennettstuart** [comment]
Build failures are not related to the changes in this PR
`dotnet.exe failed with code 1` on Test step of pipeline

**Comment by benoutram** [comment]
> Build failures are not related to the changes in this PR
> `dotnet.exe failed with code 1` on Test step of pipeline

I've merged the fix into this branch now 👍

**Pull request merged**